src/cgi.h

```diff
@@ -49,8 +49,8 @@ extern int cgi_display_errors;
 extern void cgi_init_headers(void);
 extern void cgi_redirect(char *url);
 extern void cgi_fatal(const char *error);
```

Comment:
I think as long as there's no stable API with proper soname versioning, it's not that important. But it should be considered if anyone resumes maintaining this piece of software.